src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -72,29 +60,15 @@ public async Task<ActionResult<OrderModel>> GetAsync(CallOffId callOffId)
         [TypeFilter(typeof(OrganisationIdOrganisationAuthorizationFilter))]
         public async Task<ActionResult<IList<OrderListItemModel>>> GetAllAsync(Guid organisationId)
         {
-            return await context.OrderingParty
-                .Where(o => o.Id == organisationId)
-                .SelectMany(o => o.Orders)
-                .Select(o => new OrderListItemModel(o))
-                .AsNoTracking()
-                .ToListAsync();
+            var orderingList = await orderService.GetOrderList(organisationId);
```

Comment:
I think something like `allOrders` or `orders` might be a better name than `orderingList`.

Reply:
Updated variable name to `orders `